renovate.json

```diff
@@ -1,18 +1,20 @@
 {
   "extends": [
-    "github>openstack-k8s-operators/renovate-config:default.json5"
+    "github>openstack-k8s-operators/renovate-config:stable.json5"
```

Comment:
Defined in https://github.com/openstack-k8s-operators/renovate-config/pull/24 in a way that it only allows bump of deps within the openstack-k8s-operators org.